### PR TITLE
New liquid: Poisonous Water

### PR DESCRIPTION
**Pull request by mrtechtroid**
- [x] This PR has been tested locally
- Two damage per second. (changable to 1 hp/sec)
- Greenish tint.
- Opinion Required: Whether to use Minetest ABM instead of damage_per_second parameter, where any player in a one-block-radius of the poisonous water, causes the player to get damaged. Like lets say a person is standing on a cobblestone, around which posinous water is surrounded, then he will also get a damage.

### CODE REVIEW

File: mods/other/poison_water/mod.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name=poison_water
+description = Adds poisonous water and a poisonous water bucket.
+depends = default, bucket
+optional_depends =
```
Comment (farooqkz) [inline mods/other/poison_water/mod.conf:4]:
```suggestion
depends = default
optional_depends = bucket
```

Comment (mrtechtroid) [reply mods/other/poison_water/mod.conf:4]: Hi bucket is a necessary depend, else the code crashes.

Comment (farooqkz) [reply mods/other/poison_water/mod.conf:4]: hmm up to you. but you could make it optional using an `if`

Comment (mrtechtroid) [reply mods/other/poison_water/mod.conf:4]: Tried with an if and it caused a crash, thats why had to make it a depends.

Comment (mrtechtroid) [reply mods/other/poison_water/mod.conf:4]: The same reason is causing the Automatic Lua check to fail, since its not using the depends properly.

**Pull request merged**